src/app/faq/page.tsx

```diff
@@ -9,7 +9,62 @@ export const metadata: Metadata = {
 export default function Faq() {
   return (
     <div>
-      <h1>FAQ</h1>
+      <h1>よくある質問</h1>
+      <p>
+        もしほかにご質問があれば
+        <a href='https://twitter.com/Maximum03400346' target='_blank'>
+          X(旧Twitter)
+        </a>
+        のDMなどで気軽に聞いてください！
+      </p>
+      <p>Q. プログラミング初心者なんですが、大丈夫ですか？ </p>
+      <p>
+        A.
+        Maximumではプログラミングの初歩を学ぶ初心者向け講習会を定期的に行っていますので、そちらに参加していただければ問題ありません！
+      </p>
+      <p>Q. 時間が取れないのですが、大丈夫ですか？</p>
+      <p>
+        A.
+        会員が一同に会する機会は多くなく、連絡は基本的にDiscordで行っています。そのため、講習会に毎回参加することはできなくても、討論に参加したり、遅れを取り戻すことはできます。
+      </p>
+      <p>Q. 活動の成果は？</p>
+      {/* TODO: Achievementの名前、後で決める。+ ref:https://github.com/saitamau-maximum/public-website/pull/25 */}
+      <p>A. 上のAchievementsページから過去の成績をご覧ください！</p>
+      <p>Q. 活動内容を教えてください！ </p>
+      <p>
+        A.
+        競技プログラミングやWeb制作・Webパフォーマンスチューニングなどを行っています！詳細は上のAboutページをご覧ください！
+      </p>
+      <p>Q. 会費はいくらかかるのでしょうか？ </p>
+      <p>
+        A. 月額 250 円で、入会時点での残りの今年度の活動月数分を一括徴収します。
+        例えば、 4 月に入会した場合は 3,000 円、 5 月に入会した場合は 2,750
```

Comment:
言っておいて申し訳ないけど何か月分かあるとよさそうに思ったのでお願いします :pray:

```suggestion
        例えば、 4 月に入会した場合は 12 か月分の 3,000 円、 5 月に入会した場合は 11 か月分の 2,750
```